websockets/README.md

```diff
@@ -31,10 +31,48 @@ Field Name | Type | Description
 <a name="operationBindingObjectMethod"></a>`method` | string | The HTTP method to use when establishing the connection. Its value MUST be either `GET` or `POST`.
 <a name="operationBindingObjectQuery"></a>`query` | [Schema Object][schemaObject] | A Schema object containing the definitions for each query parameter. This schema MUST be of type `object` and have a `properties` key.
 <a name="operationBindingObjectHeaders"></a>`headers` | [Schema Object][schemaObject] | A Schema object containing the definitions of the HTTP headers to use when establishing the connection. This schema MUST be of type `object` and have a `properties` key.
+<a name="operationBindingObjectQueryValues"></a>`queryValues`| object| An object containing query values.
+<a name="operationBindingObjectHeaderValues"></a>`headerValues`| object | An object containing header values
 <a name="operationBindingObjectBindingVersion"></a>`bindingVersion` | string | The version of this binding. If omitted, "latest" MUST be assumed.
 
 This object MUST contain only the properties defined above.
 
+
+## Example
+
+```yaml
+channels:
+    /employees:
+        bindings:
+            ws:
+                bindingVersion: 0.2.0
+                headers:
+                    type: object
+                    required: [
+                        'Authorization'
+                    ]
+                    properties:
+                        Authorization:
+                            type: string
+                query:
+                    type: object
+                    requried: [
+                        'employeeId'
+                    ]
+                    properties:
+                        employeeId: string
+                headerValues:
+                    Authentication: 'bearer $TOKEN'
```

Comment:
I think we should explain this `$TOKEN` syntax somewhere in this document. Otherwise, it looks like it's an example and that you actually have to put your token here.